OpenSubtitlesHandler/SnakeCaseNamingPolicy.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Linq;
+using System.Text.Json;
+
+namespace OpenSubtitlesHandler
+{
+    public class SnakeCaseNamingPolicy : JsonNamingPolicy
```

Comment:
Instead of creating a new policy I would prefer to add attributes on the properties

Reply:
Done in d82fb3f72a7f675def39de0bc521b613f1f232ae